corehq/celery_monitoring/signals.py

```diff
@@ -26,12 +30,13 @@ def stop_and_pop_timing(self):
 
         This helps avoid double-recording timings (for example when a task is retried).
         """
-        try:
-            return datetime.datetime.utcnow() - cache.get(self._cache_key)
-        except TypeError:
-            return None
-        finally:
-            cache.delete(self._cache_key)
+        time_sent = cache.get(self._cache_key)
+        if time_sent is None:
+            raise TimingNotAvailable()
```

Comment:
Did you forget to handle this error below?

https://github.com/dimagi/commcare-hq/blob/d6ca42a4b5e883da2c1b411cb56430304a601596/corehq/celery_monitoring/signals.py#L52

Reply:
Oh wow I sure did @millerdev, thanks

Reply:
Fixed in new commit